### ISSUE TITLE
Readme Update

### ISSUE THREAD

**Issue by MrHamel** [bgp/bgpq4#21]
> NOTES ON ULTRA-LARGE PREFIX-LISTS

Do we know if anyone actually tune their buffers for just a single application?

> Please note that generated prefix-lists may not fit your router's limitations. For example, JunOS supports only 85,325 prefixes in each prefix-list 4.

Would it be possible to have the program be aware of the number of prefixes when it's generating for JunOS, and throw an error + exit when the above limit is reached, or splitting up the output into multiple terms?

> SEE ALSO
> Routing Arbiter
> Cisco documentation for information on Cisco implementation of ASN32.

Is it still relevant to link off to RADB and Cisco documentation about 4-byte ASNs? At the minimum everyone and their dog has a 4-byte ASN now.

**Comment by job** [comment]
Yes, folks have tuned buffers for this one app. I've done it a number of times.

I wonder if the Junos limit still exists in modern junos. We need to test.

Agreed on those outdated links, they can be removed. Please create a pull request!

**Comment by job** [comment]
`SEE ALSO` was changed in https://github.com/bgp/bgpq4/commit/b8e92b54560430f037ed540616e0faeb3c488f2a
